spec/support/sequel.rb

```diff
@@ -19,8 +19,6 @@ class ParentSequelModel < Sequel::Model
 
     one_to_many :child_sequel_models
 
-    strict_param_setting = true
```

Comment:
It looks like it could be set globally for sequel though. Search for `strict_param_setting` on that page and you'll see it.

http://sequel.jeremyevans.net/rdoc/files/doc/mass_assignment_rdoc.html